react/components/FilterNavigator/legacy/hooks/useSelectedFilters.js

```diff
@@ -4,7 +4,7 @@ import slugify from 'slugify'
 
 import QueryContext from '../../../QueryContext'
 
-function Slugify(str) {
+export function Slugify(str) {
```

Comment:
Could you rename this function so that it's `slugify` instead of `Slugify`?
Or maybe `generateSlug` or something along those lines, so that this function doesn't shadow the `slugify` function coming from the library